DocumentedExamples/True_Zonal_Mean.ipynb

```diff
@@ -6,12 +6,11 @@
    "source": [
     "# True Zonal Mean\n",
     "\n",
-    "Calculate the \"true zonal mean\" of a scalar quantity regardless of the horizontal mesh. Specifically calcualte the volume weighted mean along all grid cells whose centres fall within finite latitude intervals rather than the arithmetic mean of cells along the model's curvilinear grid. The method presented can also be used to re-grid models onto the same latitudinal grid and the general principles can be used to define any multidimensional sum or average using the `xhistogram` package.\n",
+    "Calculate the *true zonal mean* of a scalar quantity regardless of the horizontal mesh. Specifically calcualte the volume weighted mean along all grid cells whose centres fall within finite latitude intervals rather than the arithmetic mean of cells along the model's curvilinear grid. The method presented can also be used to re-grid models onto the same latitudinal grid and the general principles can be used to define any multidimensional sum or average using the `xhistogram` package.\n",
     "    \n",
```

Comment:
This one is still too long, maybe this line needs to be actually blank? https://cosima-recipes.readthedocs.io/en/cleanupdocexamplesdescriptions/documented_examples/index.html

Reply:
done

Reply:
Speedy! Looks good now, I think?